config/actionscfg_test.go

```diff
@@ -80,6 +84,7 @@ func TestActionSCfgAsMapInterface(t *testing.T) {
 func TestActionSCfgClone(t *testing.T) {
 	ban := &ActionSCfg{
 		Enabled:             true,
+		Tenants:             &[]string{"cgrates.org", "cgrates.com"},
```

Comment:
Please replace with "itsyscom.com" as tenant, to avoid conflicts with default one.